examples/mraid/inabox-mraid.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html ⚡4ads>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp4ads-boilerplate>body{visibility:hidden}</style>
+  <meta name="amp4ads-id" content="vendor=doubleclick,type=impression-id,value=12345">
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <script async custom-element="amp-mraid" src="https://cdn.ampproject.org/v0/amp-mraid-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- <script async src="https://cdn.ampproject.org/amp4ads-v0.js"></script> -->
+  <style amp-custom>
+    .left {
+      text-align: left;
+    }
+    .right {
+      text-align: right;
+    }
+  </style>
+</head>
+<body>
+  <div class="left">
+    <amp-img id="img-1" width="150" height="100"
+        src="https://lh3.googleusercontent.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w300-h50-no">
+    </amp-img>
+  </div>
+  <div class="right">
+    <a href="https://google.com" target="_blank">
+      <amp-img id="img-2" width="150" height="100"
+          src="https://lh3.googleusercontent.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w300-h80-no">
+      </amp-img>
+    </a>
+  </div>
+  <amp-pixel src="https://www.google.com/?cid=CLIENT_ID(a)"></amp-pixel>
+  <amp-mraid layout=nodisplay></amp-mraid>
```

Comment:
we can have `amp-mraid` as a service that runs immediately after being loaded. instead of as an element waiting for runtime to schedule layout.